File: Sources/gestion-culturelle-back/src/main/java/com/afpa/cda/dto/ManifestationDto.java
package com.afpa.cda.dto;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Builder
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class ManifestationDto {
	private int id;
	private String label;
	private Date dateValidation;
	private UserDto validateur;
	private AnimationDto animation;
	private Date dateDebut;
	private Date dateFin;
	private double cout;
	private SalleDto salle;
	private double prixBillet;
	private int reservations;
	private int reservationsVip;
	
}

File: Sources/gestion-culturelle-front/project/src/app/model/manifestationDto.ts
import { SalleDto } from "./salleDto";
import { AnimationDto } from './animationDto';
import { User } from './user';

export class ManifestationDto {
    id: number;
    label: string;
    dateValidation: Date;
    validateur: User;
    animation: AnimationDto;
    dateDebut: Date;
    dateFin: Date;
    cout: number;
    salle: SalleDto;
    prixBillet: number;
    reservations: number;
    reservationsVip: number;
  //  listVips : User;

    constructor(id?: number,
        label?: string,
        dateValidation?: Date,
        validateur?: User,
        animation?: AnimationDto,
        dateDebut?: Date,
        dateFin?: Date,
        cout?: number,
        reservations?: number,
        reservationsVip?: number,
        prixBillet?: number,
        salle?: SalleDto,
    //    listVips?: User,
    ) {
        this.id = id;
        this.label = label;
        this.dateValidation = dateValidation;
        this.validateur = validateur;
        this.animation = animation;
        this.dateDebut = dateDebut;
        this.dateFin = dateFin;
        this.cout = cout;
        this.prixBillet = prixBillet;
        this.reservations = reservations;
        this.reservationsVip = reservationsVip;
        this.salle = salle;
      //  this.listVips = listVips;

    }
}


